apps/shipments/models/shipment.py

```diff
@@ -496,7 +498,7 @@ def pick_up(self, document_id=None, asset_physical_id=None, **kwargs):
 
         if asset_physical_id:
             self.asset_physical_id = hashlib.sha256(asset_physical_id.encode()).hexdigest()
-            self.validate_gtx()
+            celery.current_app.send_task('apps.shipments.tasks.gtx_validation_task', task_id=self.id, countdown=60)
```

Comment:
Why are we setting countdown=60 here? Any reason why we don't want the validation to take place ASAP?

Reply:
Also task_id should not be set to Shipment id, since that would not be unique to all tasks. Must be unique or undefined.

Reply:
I set the base countdown to 60 since I wanted to ensure that it was happening outside the original call and wouldn't delay it, can take it out.

Reply:
👍 celery tasks execute in a different process. In this case, they execute in a completely different docker container. You can remove that delay